### ISSUE TITLE
Notice: Introducing Dummy - Our New Collaborator Bot

### ISSUE THREAD

**Issue by KartSriv** [KartSriv/Story#27]
@Hampusm I want to introduce you to our new **collaborator** @TheStoryDummy! **He is not a human but he is a bot**. He runs on my **Google Cloud Server** and will manage all the story PRs from now on. He is still **buggy** so he'll start serving from next week [hopefully]! This is @TheStoryDummy's **specs**.
![image](https://user-images.githubusercontent.com/25691607/56346321-7761a500-61df-11e9-8d1d-71b8db94fe89.png)

**I have enough money to run him for a year but later we need donations**

![image](https://user-images.githubusercontent.com/25691607/56346435-b1cb4200-61df-11e9-8cbd-f1c42dd51692.png)

**This is what he'll be running**

![image](https://user-images.githubusercontent.com/25691607/56346475-c3ace500-61df-11e9-94e1-a88727e53aac.png)

**Comment by KartSriv** [comment]
He is ALIVE!

![image](https://user-images.githubusercontent.com/25691607/56346645-3e760000-61e0-11e9-86a8-30d4cd85f7be.png)

**Comment by Hampusm** [comment]
Duuuude youre making this too overly complicated. The bot doesnt have to run on some pay to use server. Its easier and less expensive to just run the bot on your own server

**Comment by Hampusm** [comment]
I could show you if you give me a access token for @TheStoryDummy

**Comment by KartSriv** [comment]
@Hampusm I want to make it trouble-free. It costs me literally nothing cause I have 200$ of Cloud Credit so let just leave it. Thanks - Karthik Srivijay

**Comment by Hampusm** [comment]
And in case you still want it on a server, i got my own personal physical server which i use for files rn but i can add another port as well

**Comment by Hampusm** [comment]
@KartSriv yeah it costs you nothing rn but what about after those 200$ have been used. Wouldnt it be better with a long term solution

**Comment by KartSriv** [comment]
@KartSriv Maybe I'll later move to AWS cause my new school has AWS educate plans.
